### ISSUE TITLE
gedit failed to build on 10.11

### ISSUE THREAD

**Issue by apjanke** [Homebrew/homebrew-core#5543]
Build is failing with this error.

```
(cd . && /usr/local/opt/glib/bin/glib-mkenums --template gedit/gedit-enum-types.h.template gedit/gedit-notebook.h gedit/gedit-app-activatable.h gedit/gedit-app.h gedit/gedit-commands.h gedit/gedit-debug.h gedit/gedit-document.h gedit/gedit-encodings-combo-box.h gedit/gedit-menu-extension.h gedit/gedit-message-bus.h gedit/gedit-message.h gedit/gedit-progress-info-bar.h gedit/gedit-statusbar.h gedit/gedit-tab.h gedit/gedit-utils.h gedit/gedit-view-activatable.h gedit/gedit-view.h gedit/gedit-window-activatable.h gedit/gedit-window.h) > gedit/gedit-enum-types.h
/usr/local/opt/glib/bin/glib-compile-resources				\
		--target="gedit/gedit-resources.c"						\
		--sourcedir="./gedit/resources"			\
		--sourcedir="./gedit/resources"		\
		--generate-source "./gedit/resources/gedit.gresource.xml"
failed to load "./gedit/resources/pixmaps/gedit-logo.png": Couldn't recognize the image file format for file './gedit/resources/pixmaps/gedit-logo.png'
./gedit/resources/gedit.gresource.xml: Child process exited with code 1.
make: *** [gedit/gedit-resources.c] Error 1
```

Logs [here](https://gist.github.com/cce31d2f07cef02c3161b3bae78a2e66).

**Comment by ilovezfs** [comment]
gdk-pixbuf just needs to be be revision bumped. Its bottle is bad due to the (fixed) etc/var bug from a week ago.

**Comment by apjanke** [comment]
Looks like that's it. Rebuilding `gdk-pixbuf` from source locally fixed this for me. Submitted a revision bump at #5545.

**Comment by ilovezfs** [comment]
Thanks, Andrew. Sorry for not taking care of this earlier :)

**Comment by apjanke** [comment]
Fixed by #5545.
